utils/text_formatting.jsx

```diff
@@ -257,6 +257,14 @@ export function escapeHtml(text) {
     return text.replace(/[&<>"']/g, (match) => htmlEntities[match]);
 }
 
+export function convertEntityToCharacter(text) {
+    return text.replace(/&amp;/g, '&').
```

Comment:
Turning `&amp;` into `&` should probably happen last. Since it's happening first, `&amp;lt;` would get turned into `&lt;` and then `<`

Reply:
Oh, you're right.  Thanks!
Now updated the order with test.